concordion/src/main/java/eu/drus/jpa/unit/api/concordion/JpaUnitConcordionRunner.java

```diff
@@ -18,13 +29,14 @@
 
 import eu.drus.jpa.unit.api.JpaUnitException;
 import eu.drus.jpa.unit.concordion.ConcordionInterceptor;
+import eu.drus.jpa.unit.concordion.EnhancedProxy;
 import eu.drus.jpa.unit.concordion.EqualsInterceptor;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'eu.drus.jpa.unit.concordion.EqualsInterceptor'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)